### DISCUSSION TITLE
How to allow word break on the Tabs label?

### DISCUSSION THREAD

**Discussion by dmdum** [ant-design/ant-design#42404] [Q&A] [answered]
I would like to limit the width of each tab item labels. 
I tried overriding the `.ant-tabs-tab-btn` class to set the `width, max-width, and overflow-wrap`, but nothing is happening.

What class should I manipulate in order to allow a word break when an overflow occurs with the tab labels? thanks!

**Comment by zombieJ** [comment, marked as answer]
https://codesandbox.io/s/ji-ben-antd-5-4-5-forked-9ujb2h?file=/index.css

PS: Not sure this looks good~

**Reply by dmdum** [reply]
That works. Thank you!

**Reply by dmdum** [reply]
@zombieJ is tabs readily responsive when placed inside a `Content` component? We follow the structure in the `Layout` component example (with the `Header` and `Sidebar`). When I change to a tablet view, the `Tabs` component seem to keep its original width unlike the other components like `Collapsible`. 

But maybe you already have a demo for this scenario? I'm trying to find the example website where the different Components are put together but I can't seem to find it...

**Reply by dmdum** [reply]
nvm, it was actually just in conflict with one of my styles
